### DISCUSSION TITLE
Delay starting the print - Is it possible?

### DISCUSSION THREAD

**Discussion by Eloque** [sn4k3/UVtools#807] [Q&A]
I've tot a AnyCubic Mono4k. Due to the noise the printer makes, I would like to start it in the middle of the night, when I am sleeping. Kind of like those washing machines that you can delay start.

It uses PWMA files, that UVTools can edit it seems.

Would this be possible? It would in essence be a delay or pause item before the first layer starts. But I can't really find how to that, I would say via scripting perhaps?

**Comment by sn4k3** [comment]
Not possible.

**Comment by Eloque** [comment]
Okay, but why?

There is no way to manipulate layer 1 to have a 3.600 second delay before starting or such? Or having a layer 1 that is not a real layer sits at Z=40 and has no exposure, a 3.600 second off cycle prepended to an PWMA file?

In FDM printers I could do prepend a gcode `G4 P60000` type thing to just wait 60 seconds. Nothing like that possible?

**Reply by sn4k3** [reply]
Anycubic file format does not allow any of the strategies you are asking for. Is one of the worse out there.
Pure gcode printers can do it, CTB also

**Comment by Hazoooka** [comment]
By noise you mean stepper motor noise or the cooling fan?
Phrozen Mini 8k is fairly quiet comparing to it’s cooling fan which is quite loud, not a good sleep having it running near. 
If this is the case I do not think usual layer delay will help. 
there is no ‘stand by’ mode in this printer I believe, so you will have to start it anyway so it will be loud all the way
